packages/components/src/tooltip/index.js

```diff
@@ -89,6 +90,11 @@ class Tooltip extends Component {
 				return;
 			}
 
+			// If pressed key is escape, no further actions are needed.
```

Comment:
I'm a bit confused by this. What actions have been taken at this point in the event handler when pressing <kbd>Escape</kbd>?